webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/AssetExtractionService.java

```diff
@@ -165,9 +167,17 @@ public class AssetExtractionService {
 
   @Autowired LocalBranchToEntityBranchConverter localBranchToEntityBranchConverter;
 
+  private RepositoryStatisticsJobScheduler repositoryStatisticsJobScheduler;
```

Comment:
You should be able to use `@Autowired` here directly?

Reply:
Please, read comment https://github.com/pinterest/mojito/pull/116#discussion_r1575394017